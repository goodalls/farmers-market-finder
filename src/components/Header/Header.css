@import url("https://fonts.googleapis.com/css?family=Open+Sans:400,700,800");
#header {
  grid-column: 1/6;
  grid-row: 1;
  font-family: "Open Sans", sans-serif;
  font-weight: 800;
  height: 20vh; }

#header h1 {
  font-size: 3em;
  color: black; }

#header span {
  display: block; }

.farmers {
  color: #D97426;
  margin-left: 5%; }

.market {
  color: #D97426;
  margin-left: 13%; }

.finder {
  color: #D97426;
  margin-left: 21%; }
