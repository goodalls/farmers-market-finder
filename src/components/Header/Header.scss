@import '../../styles/colors.scss';

#header {
  grid-column: 1/6;
  grid-row: 1;
  font-family: $font-stack;
  font-weight: 800;
  height: 20vh;
}

#header h1 {
  font-size: 3em;
  color: black;
}

#header span {
  display: block;
  
}

.farmers {
  color: $primary-orange;
  margin-left: 5%; 
}

.market {
  color: $primary-orange;
  margin-left: 13%; 
}
 
.finder {
  color: $primary-orange;
  margin-left: 21%; 
}

