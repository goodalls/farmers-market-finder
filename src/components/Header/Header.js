import React from 'react';
import './Header.css';
import { NavLink } from 'react-router-dom';

const Header = () => {
  return (
    <header id="header">
      <NavLink to={'/'}>
        <h1>
          <span className="farmers">Farmers</span>
          <span className="market">Market</span>
          <span className="finder">Finder</span>
        </h1>
      </NavLink>
    </header>
  );
};

export default Header;
