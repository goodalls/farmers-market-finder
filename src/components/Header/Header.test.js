import Header from './Header';
import {shallow} from 'enzyme';

describe('HEADER', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot();
  });
});
