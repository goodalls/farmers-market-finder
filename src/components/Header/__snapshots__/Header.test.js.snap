// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HEADER should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <NavLink
        activeClassName="active"
        ariaCurrent="true"
        to="/"
>
        <h1>
                <span
                        className="farmers"
                >
                        Farmers
                </span>
                <span
                        className="market"
                >
                        Market
                </span>
                <span
                        className="finder"
                >
                        Finder
                </span>
        </h1>
</NavLink>,
      "id": "header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "activeClassName": "active",
        "ariaCurrent": "true",
        "children": <h1>
          <span
                    className="farmers"
          >
                    Farmers
          </span>
          <span
                    className="market"
          >
                    Market
          </span>
          <span
                    className="finder"
          >
                    Finder
          </span>
</h1>,
        "to": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <span
              className="farmers"
>
              Farmers
</span>,
            <span
              className="market"
>
              Market
</span>,
            <span
              className="finder"
>
              Finder
</span>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Farmers",
              "className": "farmers",
            },
            "ref": null,
            "rendered": "Farmers",
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Market",
              "className": "market",
            },
            "ref": null,
            "rendered": "Market",
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Finder",
              "className": "finder",
            },
            "ref": null,
            "rendered": "Finder",
            "type": "span",
          },
        ],
        "type": "h1",
      },
      "type": [Function],
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/"
>
          <h1>
                    <span
                              className="farmers"
                    >
                              Farmers
                    </span>
                    <span
                              className="market"
                    >
                              Market
                    </span>
                    <span
                              className="finder"
                    >
                              Finder
                    </span>
          </h1>
</NavLink>,
        "id": "header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": <h1>
            <span
                        className="farmers"
            >
                        Farmers
            </span>
            <span
                        className="market"
            >
                        Market
            </span>
            <span
                        className="finder"
            >
                        Finder
            </span>
</h1>,
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <span
                className="farmers"
>
                Farmers
</span>,
              <span
                className="market"
>
                Market
</span>,
              <span
                className="finder"
>
                Finder
</span>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Farmers",
                "className": "farmers",
              },
              "ref": null,
              "rendered": "Farmers",
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Market",
                "className": "market",
              },
              "ref": null,
              "rendered": "Market",
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Finder",
                "className": "finder",
              },
              "ref": null,
              "rendered": "Finder",
              "type": "span",
            },
          ],
          "type": "h1",
        },
        "type": [Function],
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
